### ISSUE TITLE
Text contrast issues for Basic Page regions with colored backgrounds

### ISSUE THREAD

**Issue by jnicholCU** [CuBoulder/tiamat-theme#615]
Need to fix up more contrast and color issues with text and backgrounds.
Buttons, overlays, frames, section backgrounds, etc are still conflicting in some combinations to where text on background is not readable. Below is an example. 
![image](https://github.com/CuBoulder/tiamat-theme/assets/94021017/aae7ae67-2ec7-4fd3-ae78-3057a9c7154d)

**Comment by timurtripp** [comment]
The blue links on gold are failing the Firefox WCAG contrast test. What color are they supposed to be instead of blue?
<img width="1606" alt="Screenshot 2024-01-22 at 11 00 37 AM" src="https://github.com/CuBoulder/tiamat-theme/assets/22628823/cb22b69d-a534-44a2-a6bd-d06b35c8c2be">
